kotlin/build.gradle

```diff
@@ -41,6 +41,7 @@ buildscript {
       'mockito': '2.7.5',
       'mockitoKotlin': '1.5.0',
       'rxjava2Extensions': '0.20.4',
+      'truth': '1.0-rc1',
```

Comment:
Been meaning to switch to this. Might as well start using it for new code, I'll migrate the rest of our stuff over in a bit.

Reply:
Filed as #436